### PR TITLE
strategy part 4

### PR DESCRIPTION
**Pull request by chrisgoutzi**
Related issues: [#33]

## Description:

I added Python Basics and Power BI topics from:
[Strat Topics 4](https://docs.google.com/document/d/1N7uM-yVJr61IXV_TWmo_pxpMqZDTe3OeOyF1TqaB7xI/edit#)
to the topics/strategy folder.

I added the frontmatter for the topics as well.

## I solemnly swear that:

- [X] I ran the hugo server and looked at my changed in the browser with my own eyes
- [X] I ran the linter and there were no errors
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code

### CODE REVIEW

File: content/topics/strategy/power-bi/_index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+content_type: topic
+ready: true
+title: Power BI
+---
+
+## Power BI
```
Comment (ry-oc) [inline content/topics/strategy/power-bi/_index.md:7]: Please remove the heading as one is auto generated from the title

File: content/topics/strategy/python-basics/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+content_type: topic
+ready: true
+title: Python Basics
+---
+
+## Python Basics:
```
Comment (ry-oc) [inline content/topics/strategy/python-basics/_index.md:7]: Please remove the heading as one is auto generated from the title

**Review by ry-oc — CHANGES_REQUESTED** [review]

File: content/topics/strategy/power-bi/_index.md (hunk 1 above)
Comment (sheenarbw) [reply content/topics/strategy/power-bi/_index.md:7]: @chrisgoutzi Hey, can you please polish this off? You're rreally close to being complete.

When hugo renders this thing then the html will ook like this:

```
<h1>Power BI</h1>
<h2>Power BI</h2>
```
As you can see this is redundant and also not dry and also redundant.

Please run the application and see for yourself. The best thing to do is remove the line that says `## Power BI`

@asradebe can you please follow upi on this?

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Hey. This PR is getting quite old. Are you still planning to finish it? We appreciate the help but we might need to give this issue to someone else now.

File: content/topics/strategy/power-bi/_index.md (hunk 1 above)
Comment (asradebe) [reply content/topics/strategy/power-bi/_index.md:7]: will do

Comment (sheenarbw) [reply content/topics/strategy/power-bi/_index.md:7]: @asradebe  should we close this PR? It looks like the contributor isn't interested in finishing

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Change requests ignored?

**Comment by asradebe** [comment]
> Change requests ignored?

Closing off this PR

**Review by asradebe — COMMENTED** [review]
Hi Chris. 

I am closing off this PR as you have not attended to the requested changes and the PR has been open for weeks.


To contribute in the near future, Please join our discord channel https://discord.com/channels/758708091583332382/762669571852468254 and we will help you. You will be unassigned to the issue you were working on and this PR will be regarded as unsuccessful. Thank you for your dedication and collaboration. We appreciate the efforts.

Our sincerest thanks
The Team

**Pull request closed without merging**